### PR TITLE
Plans: Update descriptions and pricing styling for Jetpack

### PR DESCRIPTION
**Pull request by keoshi**
#### Changes proposed in this Pull Request

* Updates Jetpack plans tables according to the latest pattern in Calypso; adds audience highlight.
* Updates pricing section in signup plans tables.

#### Before, Calypso > Plans

image

#### After, Calypso > Plans

image

#### Before, JP Connect > Store

image

#### After, JP Connect > Store

image

#### Testing instructions

* Fire up this PR.
* Visit `http://calypso.localhost:3000/plans/SITE_URL` and ensure the audience highlight is visible (on both mobile and desktop).
* Visit `http://calypso.localhost:3000/jetpack/connect/store/` and ensure the pricing section looks good on desktop (mobile is unchanged).

### CODE REVIEW

File: client/lib/plans/plans-list.js [hunk 1]
```diff
@@ -666,7 +666,7 @@ export const PLANS_LIST = {
 		group: constants.GROUP_JETPACK,
 		type: constants.TYPE_FREE,
 		getTitle: () => i18n.translate( 'Free' ),
-		getAudience: () => i18n.translate( 'Best for students' ),
+		getAudience: () => i18n.translate( 'Best for Students' ),
```
Comment (a8ci18n) [inline client/lib/plans/plans-list.js]: :warning: This change will be queued for retranslation. We'll display the English text until then.

**Comment by folletto** [comment]
This looks like a good change.

One question: "Hobbyist" to me sounds dismissive. If I'm a freelance professional, and I see that, I wouldn't see that fitting. Then I see "Small Business", and again, not me. 

Is there a specific reason for that choice of word (i.e. we are already using it)?

**Comment by keoshi** [comment]
> One question: "Hobbyist" to me sounds dismissive. If I'm a freelance professional, and I see that, I wouldn't see that fitting. Then I see "Small Business", and again, not me.

I don't disagree here. We were already using elsewhere, but what do you think about “Best for Personal Use“?

**Comment by folletto** [comment]
If we are using that elsewhere then probably best keeping.

But yes, "Best for Personal Use" seem a better phrasing.

**Comment by keoshi** [comment]
@folletto Cool, let's go with "Best for Personal Use" and I'll update the stray “hobbyist” one.

**Comment by jeffgolenski** [comment]
Agree with Foletto. Nice work here!

File: client/lib/plans/plans-list.js [hunk 2]
```diff
@@ -794,8 +802,16 @@ export const PLANS_LIST = {
 			),
 		getDescription: () =>
 			i18n.translate(
-				'Comprehensive, automated scanning for security vulnerabilities, ' +
-					'fast video hosting, and marketing automation.'
+				'{{strong}}Best for Small Businesses:{{/strong}}' +
+					'Comprehensive, automated scanning for security vulnerabilities, ' +
+					'fast video hosting, and marketing automation.',
+				{
+					components: {
```
Comment (enejb) [inline client/lib/plans/plans-list.js]: Looks like we are repeating this component in a bunch of places. 
We can make it a constant and pass in the constant into the `i18n.translate( text, translationConstant )`

where
```
const translationComponents = {
					components: {
Attach files by dragging & dropping, selecting or pasting them.
						strong: (
							<strong className="plans__features plan-features__targeted-description-heading" />
						),
					};
```

**Pull request merged**